addOns/client/src/main/java/org/zaproxy/addon/client/ClientIntegrationAPI.java

```diff
@@ -138,6 +139,17 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
                 this.extension.addReportedObject(new ReportedEvent(json));
                 break;
 
+            case ACTION_REPORT_ZEST_SCRIPT:
```

Comment:
This was not added to the API, was that on purpose?

Reply:
Do we need to differentiate? Seems odd not add it to the API but should be fine.

Reply:
Yes, else we will need to parse the received object to check which kind of statement it is. So I think it will be better to keep it separate for zest statements.

Reply:
To be clear what I'm saying is that the action is not being added to the API, i.e.: https://github.com/zaproxy/zap-extensions/blob/7af0ae615bebf100ccc581e1d589259a3819d7f1/addOns/client/src/main/java/org/zaproxy/addon/client/ClientIntegrationAPI.java#L58-L59

Reply:
Oh okay. Got it.